### DISCUSSION TITLE
Error when running the code

### DISCUSSION THREAD

**Discussion by lv333ming** [nishad2m8/BTC-Ticker#4] [General]
I got the following prompt during the build process.

![image](https://github.com/user-attachments/assets/6ce115cb-68e4-418d-80c3-a983fa74dad7)

This is the prompt after build

![image](https://github.com/user-attachments/assets/e3df7b65-312e-4b6b-82cd-fa3ac5f9f6f7)

**Comment by nishad2m8** [comment]
Hi, based on the screenshot platformio.ini things are messed. 

```c


[platformio]

default_envs = T-Display-AMOLED


;! Don't make changes
boards_dir = boards

[env]
lib_ignore = lib_deps
platform = espressif32@6.7.0
...[...]
```

clone again and try to flash it

**Comment by lv333ming** [comment]
![image](https://github.com/user-attachments/assets/bcc6ca32-361c-4514-a95d-a7dc9c647c5e)
In this case, do I need to put the Arduino library file over?

**Reply by nishad2m8** [reply]
No need to add any library dependencies manually, they are already specified in the `platformio.ini` file. However, make sure to open the folder in PlatformIO, not in ESP-IDF.

```c
lib_deps = 
    lvgl/lvgl @ ~8.3.11
    lewisxhe/XPowersLib @ 0.2.1
    lewisxhe/SensorLib @ 0.2.0

    ArduinoJson
    Time
    Timezone
    WebSockets
    WiFiProv
```

<img width="1668" alt="Screenshot 2024-09-17 at 3 44 02 PM" src="https://github.com/user-attachments/assets/a8e12e62-5cd4-45a9-b793-2c5baa6004a5">

<img width="1668" alt="Screenshot 2024-09-17 at 3 42 56 PM" src="https://github.com/user-attachments/assets/47ad92d7-6dc1-4438-8be6-05ad5c8e5fe0">

**Reply by lv333ming** [reply]
Thank you very much, I have built it according to your method
